src/openzaak/utils/tasks.py

```diff
@@ -0,0 +1,791 @@
+            "A validation error occurred while deserializing a "
+            f"EnkelvoudigInformtatieObject on line {row_index}: \n"
+            f"{eio_serializer.errors}"
+        )
+
+        logger.warning(error_message)
+        document_row.comment = error_message
+        document_row.processed = True
+
+        return document_row
+
+    data: dict = eio_serializer.validated_data
+
+    if document_row.uuid:
+
+        try:
+            uuid = UUID(document_row.uuid, version=4)
+        except ValueError:
+            error_message = (
+                f"Given UUID for row {row_index} is not a valid UUID (version 4)"
+            )
+
+            logger.warning(error_message)
+            document_row.comment = error_message
+            document_row.processed = True
+
+            return document_row
+
+        if document_row.uuid in existing_uuids:
+            error_message = (
+                f"UUID given on row {row_index} was already found! Not overwriting "
+                "existing EIO."
+            )
+
+            logger.warning(error_message)
+            document_row.comment = error_message
+            document_row.processed = True
+
+            return document_row
+
+        data["uuid"] = str(uuid)
+    else:
+        data["uuid"] = str(uuid4())
+
+    gegevensgroep_fields = ("ondertekening", "integriteit")
+
+    for field in gegevensgroep_fields:
+        gegevens_groep_value = data.pop(field, {})
+
+        for key, value in gegevens_groep_value.items():
+            data[f"{field}_{key}"] = value
+
+    if "vertrouwelijkheidaanduiding" not in data:
+        informatieobjecttype = data["informatieobjecttype"]
+        data[
+            "vertrouwelijkheidaanduiding"
+        ] = informatieobjecttype.vertrouwelijkheidaanduiding
+
+    instance = EnkelvoudigInformatieObject(**data)
+
+    if not instance.identificatie:
+        instance.identificatie = generate_unique_identification(
+            instance, "creatiedatum"
+        )
+
+    try:
+        instance.clean()
+    except ValidationError as e:
+        error_message = (
+            "A validation error occurred while validating a "
+            f"EnkelvoudigInformtatieObject on line {row_index}: \n"
+            f"{str(e)}"
+        )
+
+        logger.warning(error_message)
+        document_row.comment = error_message
+        document_row.processed = True
+
+        return document_row
+
+    file_path = document_row.bestandspad
+    path = Path(settings.IMPORT_DOCUMENTEN_BASE_DIR) / Path(file_path)
+
+    if not path.exists() or not path.is_file():
+        error_message = (
+            f"The given filepath {path} does not exist or is a file for "
+            f"row {row_index}"
+        )
+
+        logger.warning(error_message)
+        document_row.comment = error_message
+        document_row.processed = True
+
+        return document_row
+
+    default_dir = _get_default_path(instance.inhoud)
+    import_path = f"{default_dir}/{path.name}"
```

Comment:
`_get_default_path` returns a `Path`, so I think this should be possible:
```suggestion
    import_path = default_dir / path.name
```